src/detrend_fancy_plot.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def _fancy_plot(
+    y_original: np.ndarray | pd.DataFrame,
+    y_fitted: np.ndarray,
+    y_detrend: np.ndarray,
+    fitted_parameters: dict,
+    xticklabels: pd.core.indexes.base.Index | None = None,
+    method_name: str = "",
+) -> None:
+    """plot two graphs : the original data and its fitted trend curve ; the detrended data
```

Comment:
nit: "Plot two graphs: the original [...]"

=> uppercase letter at the start of docstring
=> no space before ":"
=> also, no space before ";"

(nit means nitpicking => le mec qui clc quoi, et du coup tu préfixes de "nit" pour dire que tu attaques sur des détails, pour ne pas contrarier l'autre. Car il faut quand même pointer tous les petits détails, ça reste important)

Reply:
done boi

Reply:
you forgot to commit boi